components/automate-ui/src/app/modules/infra-proxy/policy-groups-list/policy-groups-list.component.html

```diff
@@ -0,0 +1,14 @@
+<chef-table>
+  <chef-thead>
+    <chef-tr class="no_border_tr">
+      <chef-th class="no_border_th">Policy Group</chef-th>
```

Comment:
it's Policyfiles Group

Reply:
I had discussion with Manish. He mistakenly put 'Policyfiles Group' in design.